Header header
int64 vehicleID
float64 accmeasx
float64 accmeasy
float64 accmeasz
float64 gyromeasx
float64 gyromeasy
float64 gyromeasz